### PR TITLE
 feat(react-ui-kit): input invalid marker, links in error messages, checkmark icon, sleaker text css

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: packages/react-ui-kit/src/Form/Input.jsx [hunk 1]
```diff
@@ -27,6 +27,15 @@ const placeholderStyle = css`
   text-transform: ${props => props.placeholderTextTransform};
 `;
 
+const dotRadius = 4;
+const dotSize = dotRadius + dotRadius;
```
Comment (lipis) [inline packages/react-ui-kit/src/Form/Input.jsx]: or this.. since you are avoiding magic numbers..

```js
const dotSize = dotRadius << 1;
```

Comment (AndyLnd) [reply packages/react-ui-kit/src/Form/Input.jsx]: Maybe i will do that 😄 
But it wouldn't work with fractional numbers ...

Comment (lipis) [reply packages/react-ui-kit/src/Form/Input.jsx]: Jokes aside I prefer `<< 1` over `+` here :)

Comment (lipis) [reply packages/react-ui-kit/src/Form/Input.jsx]: also `dotDiameter` is more suitable in this case :)

File: packages/react-ui-kit/src/Icon/CheckIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ */
+
+import IconBase from './IconBase';
+import React from 'react';
+
+class CheckIcon extends IconBase {
+  /* eslint-disable no-magic-numbers */
+  width = 16;
+  height = 12;
```
Comment (lipis) [inline packages/react-ui-kit/src/Icon/CheckIcon.jsx]: in theory we need to enable it as well afterwards..

File: packages/react-ui-kit/src/Form/Input.jsx (hunk 1 above)
Comment (AndyLnd) [reply packages/react-ui-kit/src/Form/Input.jsx]: i changed everything, since svg allows me to scale the image as a whole

**Pull request merged**